docs/program/apis/sessions.md

```diff
@@ -0,0 +1,88 @@
+linkTitle: "Session Management API"
+weight: 20
+type: "docs"
+description: "How to use the Sessions Management API with Viam's Client SDKs."
+tags: ["client", "sdk", "viam-server", "networking", "apis", "robot api", "session", "sessions", "session management"]
+---
+
+The Session Management API provides support for robot session management.
+
+A Session allows a **client** to express that it is actively connected or authenticated to `viam-server` on your robot and
+supports stopping actuating components when the connection is no longer active.
+
+{{< alert title="Info" color="info" >}}
+
+### Purpose of Session Management
+
+When controlling a robot or fleet with the Viam platform, you want some kind of protocol or system-wide means to understand the presence of the clients that are talking and authenticated to each robot's `viam-server` through Viam's various [resource APIs](/program/apis/).
+
+The presence of these clients at one point in time can be referred to as a "session".
+A session technically is any presence mechanism at the RDK application layer maintained by any client with `viam-server`.
+Session management, as explained here, allows for safer robotic operation when you are dealing with moving parts on the robot (the "actuating controls").
+
+Specifically, without session management, controls that would be "sticky" (for example [`SetPower()` of a base](/components/base/#setpower)) based on the last input of a **client**, meaning the API request from the client sets the flow of electricity on a part of the robot and then does not time out, could lead to **having a robot trying to continue doing what it was told to do forever.**
+
+You do not want that to happen.
+
+#### Clients and Viam's Client SDKs
+
+A client has multiple meanings for a Viam robot.
+Essentially, it's anything that is receiving the information served by `viam-server` running on the robot.
+
+A **client** can be an SDK script controlling the robot, an input controller, or just the different resources on the robot talking amongst themselves.
+For example, if you use Viam's module registry to [add modular resources to your robot](/extend/modular-resources/), the clients of your robot in its lifetime will include the "model servers" you instantiate on your robot for individual resources.
+
+Viam's Session Management API is your built-in solution to this.
+Your client maintains the session, telling the `viam-server` instance that it is still present every so often, or staying within the heartbeat window.
+
+Your client must send at least one session heartbeat within this window.
+As soon as the window lapses/expires, the server will safely stop all resources that are marked for safety monitoring that have been last used by that session, and no others; a lapsed client will attempt to establish a new session immediately prior to the next operation it performs.
+
+{{< /alert >}}
+
+## Usage
+
+Usage of the Sessions Management API differs across [Viam's SDKS](/program/).
+
```

Comment:
```suggestion

### Manage on-robot sessions

```
Why not put your tabs in here, since it looks like this is the only part that differs between Go and Python, and then you'll get a handy heading in the right nav?

Reply:
good idea!!